libraries/lib-sentry-reporting/AnonymizedMessage.h

```diff
@@ -0,0 +1,55 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file AnonymizedMessage.h
+ @brief Declare a class to store anonymized messages.
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#include <string>
+#include <wx/string.h>
+
+#pragma once
+
+namespace audacity
+{
+namespace sentry
+{
+
+class SENTRY_REPORTING_API AnonymizedMessage final
```

Comment:
A class comment please, explaining that initialization strips paths, and then nothing changes the contents except assignments and moves from the same class.